website/content/en/docs/reference/configuration/tls.md

```diff
@@ -0,0 +1,150 @@
+---
+title: TLS configuration
+short: TLS configuration
+weight: 5
+aliases: [
+  "/docs/reference/tls",
+  "/docs/reference/configuration/tls",
+]
+---
+
+Vector implements cryptography and secure communication using the [OpenSSL][openssl] library.
+In particular, the official Vector binaries are statically linked against OpenSSL version
+{{< param openssl_version >}} and do not use any OpenSSL library installed on the running system.
+
+Note that OpenSSL recognizes a number of [environment variables][openssl-env] independently of Vector.
+
+## Trusted Certificates
+
+Trusted certificates (also called certificate authorities) are used for client and server verification.
+
+By default, OpenSSL and Vector will look for trusted certificates in the following locations:
```

Comment:
```suggestion
By default, OpenSSL and Vector looks for trusted certificates in the following locations:
```

Reply:
"OpenSSL and Vector" is a plural subjct, shouldn't the verb be "look"?

Reply:
Ah, yes, you're right!

Reply:
No problem! I will rewrite it anyway in the present tense instead of future like the others.